features/admin/deactivate_supplier_user.feature

```diff
@@ -0,0 +1,26 @@
+@deactivate-supplier-user
+Feature: Deactivate a supplier's contributor
+
+Scenario Outline: Correct users can deactivate a supplier's contributor
+  Given I am logged in as the production <role> user
+  # TODO get here from /admin
+  And I am on the /admin/suppliers?supplier_name_prefix=DM+Functional+Test+Supplier page
+  And I click the summary table 'Users' link for 'DM Functional Test Supplier'
+  And I click the summary table 'Deactivate' button for 'DM Functional Test Supplier User 1'
```

Comment:
This needs a check that the user is now actually deactivated.

Reply:
So this isn't actually possible in any normal kind of way (ie checking the page for new content). None of the values in that table change. However, the activate button is only present if they have already been deactivated. So finding and clicking activate kind of serves as the test. I've put in a step to check the table content doesn't change for that user when it's not supposed to though.

Reply:
I guess the way to do it would be to try to log in with that user account and check it's blocked.  But also the fact that the "Activate" button exists is probably enough.